workflows/theiacov/wf_theiacov_clearlabs.wdl

```diff
@@ -37,6 +37,8 @@ workflow theiacov_clearlabs {
     String? target_organism
```

Comment:
should this be kraken2_target_organism? i feel like we discussed this at one point in time and cannot remember the conclusion

Reply:
There's a conflict with the output declaration in this particular case so I'm unsure how to proceed here. I0m leaving it as `target_organism` until we can discuss it further

Reply:
on theiacov_ont is also `target_organism`

Reply:
Would there be an instance we'd want this input to feed into multiple tasks, to ensure consistency? My gut reaction to having workflow level inputs be specific to tasks, is we should only do that if we could/would not use it in multiple workflows. This, to me, seems like an instance we might want to leave it without any tool name to allow for just that. Maybe worth some discussion? Ultimately, when we get to the PHB-lite effort, this will get abstracted away, but something to consider in the interim.

Reply:
Pretty sure that's what we're using the organism_parameter workflow to do in order to standardize the language here. 

Could you help me understand what you mean by having workflow level inputs specific to tasks if they're not used in multiple workflows? 

If I'm understanding correctly, then we might be better served by removing all parameters that can be modified in the organism_parameters workflow so that they're not workflow-level inputs